platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -521,13 +521,15 @@ public void onSurfaceTextureUpdated(SurfaceTexture surface) {
   @CallSuper
   protected void onDetachedFromWindow() {
     super.onDetachedFromWindow();
-    mapZoomButtonController.setVisible(false);
+    if (mapZoomButtonController != null) {
+      mapZoomButtonController.setVisible(false);
+    }
   }
 
   // Called when view is hidden and shown
   @Override
   protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
-    if (isInEditMode()) {
+    if (isInEditMode() || mapZoomButtonController == null) {
```

Comment:
just to clarify, I also completely agree if there is more to it than just null checking but this just feels weird to write/read:
```
boolean something = somehtinToTest == null;
```